### ISSUE TITLE
Document recommend approach for calling web API using PoP tokens

### ISSUE THREAD

**Issue by dapathy** [Azure-Samples/ms-identity-javascript-angular-tutorial#199]
# Issue

Please document recommended approach for calling an API with an access token acquired via proof of possession using server nonce:

[https://github.com/Azure-Samples/ms-identity-javascript-angular-tutorial/tree/main/7-AdvancedScenarios/2-call-api-pop](https://github.com/Azure-Samples/ms-identity-javascript-angular-tutorial/tree/main/7-AdvancedScenarios/2-call-api-pop)

## This issue is for the sample

```console
    - [ ] 1-1) Sign-in with Azure AD
    - [ ] 1-2) Sign-in with Azure AD B2C
    - [ ] 2-1) Acquire a Token and call Microsoft Graph
    - [ ] 3-1) Protect and call a web API on Azure AD
    - [ ] 3-2) Protect and call a web API on Azure AD B2C
    - [ ]   4) Deploy to Azure Storage and App Service
    - [ ] 5-1) Call a web API using App Roles
    - [ ] 5-2) Call a web API using Security Groups
    - [ ] 6-1) Call a multi-tenant web API
    - [ ] 7-1) Call Microsoft Graph using on-behalf-of flow
    - [x] 7-2) Call a web API using Proof of Possession tokens
```

## This issue is for a

```console
    - [ ] bug report -> please search issues before submitting
    - [ ] question
    - [ ] feature request
    - [x] documentation issue or request
```

**Comment by salman90** [comment]
Hi @dapathy, we are still working on implementing proof of possession with MSAL. The item is currently in our backlog, and once the feature is updated in MSAL, we will update the sample.

**Comment by derisen** [comment]
@dapathy apologies for the confusion. We have decided instead to archive the sample, as the proof of possession feature is not fully ready for the public consumption. If you really need, the sample is still available on [another branch](https://github.com/Azure-Samples/ms-identity-javascript-angular-tutorial/tree/pop-sample-archive), but we won't be able to update it in the foreseeable future.

Closing this.
